hybrid-array/Cargo.toml

```diff
@@ -18,3 +18,4 @@ rust-version = "1.65"
 
 [dependencies]
 typenum = "1.17"
+zeroize = { version = "1.7", path = "../zeroize" }
```

Comment:
Yeah, feature-gating is good.

```suggestion
zeroize = { version = "1.7", optional = true, default-features = false, path = "../zeroize" }
```

Reply:
Should it be default?

Reply:
We have it feature-gated elsewhere, so making it off-by-default makes it easier to map to that, although you can always do `default-features = false`